org.eclipse.jdt.ls.core/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,8 @@ Bundle-Version: 0.20.0.qualifier
 Bundle-Activator: org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Localization: plugin
-Import-Package: org.osgi.framework;version="1.3.0"
+Import-Package: org.osgi.framework;version="1.3.0",
+ org.eclipse.search.core.text
```

Comment:
I missed it, and it's not obvious but org.eclipse.search.core.text has a dependency to Eclipse UI bundles, and drags down a lot of unwanted stuff.

It brings the server size to 69MB, instead of previous 38MB.
We might need to copy the org.eclipse.search.core.text and org.eclipse.search.internal.core.text packages from org.eclipse.search

Reply:
The PR was updated with new ported classes and the description